zds/utils/tests/test_misc.py

```diff
@@ -56,3 +57,48 @@ def test_remove_url_scheme(self):
         for element in oracle:
             # as we are not in py3 we do not have subTest method. so we use a bare for loop.
             self.assertEquals(remove_url_scheme(element.given), element.expected)
+
+
+class TemplateTagsTest(TestCase):
```

Comment:
Faut surtout tester les cas particuliers qui risquent de planter, comme `joinby([])`.

Reply:
Ah oui, bien vu. Cependant, ce cas est prévu par le " if not values: return '' ". ;)